src/GitHub.VisualStudio/GitHub.VisualStudio.csproj

```diff
@@ -11,7 +11,7 @@
     <NuGetPackageImportStamp>
     </NuGetPackageImportStamp>
     <TargetFrameworkProfile />
-    <ApplicationVersion>2.3.0.16</ApplicationVersion>
```

Comment:
Why is this here? This is the release branch for 2.3.0, it should not have a version bump from master